atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraRequestExceptionHandler.java

```diff
@@ -109,11 +109,12 @@ private RequestExceptionHandlerStrategy resolveStrategy() {
     }
 
     private static AtlasDbDependencyException logAndThrowException(
-            int numberOfAttempts, Exception ex, RetryableCassandraRequest<?, ?> req) {
+            int numberOfAttempts, Exception ex, CassandraServer serverTried, RetryableCassandraRequest<?, ?> req) {
         log.warn(
                 "Tried to connect to cassandra {} times. Exception message was: {} : {}",
                 SafeArg.of("numTries", numberOfAttempts),
```

Comment:
This can be misleading. Could you _also_ add the `numberOfAttemptsOnHost` argument here? That way, we can see there are two separate numbers: one for the `serverHostNameTried` and another for the general case.